clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClient.scala

```diff
@@ -18,20 +28,28 @@ object ClioClient extends App {
     ClioClientConfig.ClioServer.clioServerPort,
     ClioClientConfig.ClioServer.clioServerUseHttps
   )
-  val client = new ClioClient(webClient)
 
-  System.exit(client.execute(args))
+  override def run(command: CommandType, remainingArgs: RemainingArgs): Unit = {
+    sys.exit(
+      new ClioClient(webClient, command, commonOptions.bearerToken).execute()
+    )
+  }
+
+  override def beforeCommand(options: CommonOptions,
```

Comment:
Nitpick: it might be clearer how things work to have this method defined above `run`